app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,39 @@
+<div class="row vh-100 d-flex justify-content-center">
+    <div class="col-md-6">
+      <div class="card">
+        <div class="card-body">
+          <h2 class="card-title text-center">Log in</h2>
+
+          <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+            <div class="form-group">
+              <%= f.label :email %><br />
+              <%= f.email_field :email, class: "form-control", autofocus: true, autocomplete: "email" %>
+            </div>
+
+            <div class="form-group">
+              <%= f.label :password %><br />
+              <%= f.password_field :password, class: "form-control", autocomplete: "current-password" %>
+            </div>
+
+            <% if devise_mapping.rememberable? %>
+              <div class="form-group">
+                <%= f.check_box :remember_me %>
+                <%= f.label :remember_me %>
+              </div>
+            <% end %>
+
+<<<<<<< HEAD
+            <div class="tex-center">
+              <%= f.submit "Log in", class: "btn btn-primary btn-lg btn-block" %>
+=======
+            <div class="form-group">
+              <%= f.submit "Log in", class: "btn btn-primary" %>
+>>>>>>> 8049a108d62aceef21f491e4887a3c47599d0ea7
+            </div>
+          <% end %>
```

Comment:
- [ Required ] As requested by the previous reviewer, I think it would be nice if you made the `Login` button have the same width as the other input elements which take the `full width` of the login form. Currently, the button has a smaller width compared to the other input elements. Please see the screenshot and code suggestion below and rectify this.

## Example code
```ruby
   <div class="form-group mb-3">
      <%= f.submit "Log in", class: "btn btn-primary w-100" %>
  </div>
```

## Log In form
![Screenshot from 2023-11-18 23-58-35](https://github.com/MenendezON/Blog-app/assets/16392046/ecfd6de7-79a4-4990-b99f-8ce39ca57bb5)